vendor/patches/eembc_coremark/0001-no-minimum-run-time.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/core_main.c b/core_main.c
+index 6161974..cca596c 100644
+--- a/core_main.c
++++ b/core_main.c
+@@ -295,10 +295,16 @@ MAIN_RETURN_TYPE main(int argc, char *argv[]) {
+ 	if (time_in_secs(total_time) > 0)
+ 		ee_printf("Iterations/Sec   : %d\n",default_num_contexts*results[0].iterations/time_in_secs(total_time));
+ #endif
+-	if (time_in_secs(total_time) < 10) {
```

Comment:
How about adding a `define` for this functionality, giving people to override it in the Makefile? If you do so, please also add this piece of information to the README.

Reply:
I think we're better off without. The reason to put this code back in is to meet the 'letter of the law' wrt to publishing coremark results (absolutely no alterations to this file amongst others). The define is still a file alteration so doesn't help you if you want 'letter of the law' coremark results.